internal/resources/slo/data_source_slo.go

```diff
@@ -212,6 +216,18 @@ func unpackAlertingMetadata(metaData slo.SloV00AlertingMetadata) []map[string]in
 	return retAlertMetaData
 }
 
+func unpackAdvancedOptions(options slo.SloV00AdvancedOptions) []map[string]int {
```

Comment:
question: Should this return type be `[]map[string]interface{}` like other `unpack...` functions?

I don't know if this matters

Reply:
Elaine told me to use this and it worked. I originally had it the interface way. I can try it the other way and see if it passes the tests?

Reply:
I'd trust Elaine. I was just seeing the bits of code above and below. more type information is better right?